app/assets/javascripts/arclight/search_results.js

```diff
@@ -0,0 +1,15 @@
+Blacklight.onLoad(function () {
+    'use strict';
```

Comment:
Can we use 2 space indention here? (sorry its not in the eslint for some reason) Its the convention in all of the other JavaScript.

Reply:
Yup! Also maybe I should add that to the eslint?

Reply:
It is in the eslint via the included `airbnb-base/legacy`. The eslint was not actually running on my machine, because when I haven't got it installed, the rake task silently fails rather than erroring. I wonder if it's installed and running on Travis.